src/psm.t.sol

```diff
@@ -35,7 +35,7 @@ interface Hevm {
 
 contract TestToken is DSToken {
 
-    constructor(bytes32 symbol_, uint256 decimals_) public DSToken(symbol_) {
+    constructor(string memory symbol_, uint8 decimals_) public DSToken(symbol_) {
```

Comment:
generally I'd prefer to keep the input param a uint256 and then check that it's LT uint8 max

Reply:
Does it really matter for the test with a hardcoded value though?

Reply:
I suppose it's fine in this case. There's not going to be much chance this is going to be deployed manually. As a general rule we probably want to stick to 32-byte data params when possible.